### PR TITLE
[ffmpeg] fix FindFFMPEG.cmake to work properly even without avresample

### PR DESCRIPTION
**Pull request by mcmtroffaes**
When building ffmpeg[core] (without avresample support), FindFFMPEG.cmake claims to not have fully found the ffmpeg libraries, because it still looks for the avresample library. This patch fixes that problem.

**Describe the pull request**

- What does your PR fix? See above.

- Which triplets are supported/not supported? Have you updated the CI baseline? Unchanged.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)? To the best of my knowledge, yes.

### CODE REVIEW

**Review by LilyWangL — APPROVED** [review]

File: ports/ffmpeg/portfile.cmake [hunk 1]
```diff
@@ -127,8 +127,10 @@ else()
     set(OPTIONS "${OPTIONS} --disable-bzlib")
 endif()
 
+set (ENABLE_AVRESAMPLE OFF)
```
Comment (strega-nil) [inline ports/ffmpeg/portfile.cmake:130]:
```suggestion
set(ENABLE_AVRESAMPLE OFF)
```

File: ports/ffmpeg/portfile.cmake [hunk 2]
```diff
@@ -127,8 +127,10 @@ else()
...
 if("avresample" IN_LIST FEATURES)
     set(OPTIONS "${OPTIONS} --enable-avresample")
+    set (ENABLE_AVRESAMPLE ON) #necessary for configuring FFMPEG CMake Module
```
Comment (strega-nil) [inline ports/ffmpeg/portfile.cmake:133]:
```suggestion
    set(ENABLE_AVRESAMPLE ON) #necessary for configuring FFMPEG CMake Module
```

**Review by strega-nil — APPROVED** [review]
Minor changes to make the cmake more consistent.

**Comment by mcmtroffaes** [comment]
Thanks for the merge!

**Pull request merged**